src/components/LeftSignInUp/LeftSignInUp.jsx

```diff
@@ -0,0 +1,54 @@
+import {
+  AiFillTwitterCircle,
+  AiFillLinkedin,
+  AiFillGithub,
+} from 'react-icons/ai';
+import { IoLogoDiscord } from 'react-icons/io5';
+
+function LeftSignInUp() {
```

Comment:
This is a confusing name for a component 🗡️  Can we change it to something more meaningful?

Reply:
I thought about it but nothing came to my mind.🤔 Do you have any suggestion?

Reply:
What about WelcomeContent ?

Reply:
AuthenticationModuleSpan might be something to go with.